hepcrawl/tohep.py

```diff
@@ -243,7 +243,7 @@ def _filter_affiliation(affiliations):
     for author in crawler_record.get('authors', []):
         builder.add_author(builder.make_author(
             full_name=author['full_name'],
-            affiliations=_filter_affiliation(author['affiliations']),
+            affiliations=_filter_affiliation(author.get('affiliations', [])),
```

Comment:
the `_filter_affiliation` thing is not needed, as the builder is already cleaning up empty values. Besides, this should be `raw_affiliations` instead of `affiliations`, see #185.